modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/dto/v1_0/converter/TaxonomyCategoryDTOConverter.java

```diff
@@ -179,6 +186,27 @@ private TaxonomyCategory _toTaxonomyCategory(
 							}
 						};
 					});
+				setPermissions(
+					() -> {
+						if (!FeatureFlagManagerUtil.isEnabled("LPD-41304")) {
+							return null;
+						}
+
+						_permissionService.checkPermission(
```

Comment:
If User has permissions to see the category, but NOT to see the permissions of the category, this will throw an exception and the result will be a 404. I think that we have to let user see the category but not the permissions, if not, we are loosing capabilities.

test case:
create vocabulary
create category
Create user (site member, only, this is important because a site member can see the category but has no permission to see the cat permissions)

- run endpoint
- result 200
- change to member site user
- run endpoint
- result 200

stop machine > enable FF

- User admin
- run endpoint
- result 200
- change to member site user
- run endpoint
- result 404